scout/server/blueprints/cases/templates/cases/gene_panel.html

```diff
@@ -98,60 +98,72 @@ <h6 class="ms-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
-        <span class="text-muted">No gene list generated</span>
-      {% endif %}
-    </div>
+      <div class="mt-3 ms-3" style="max-height:200px; overflow-y: scroll; min-height: 100px;">
+        {% if case.dynamic_gene_list %}
+          <form id="dynamicGenes" method="POST" action="{{ url_for('cases.phenotypes_actions', institute_id=institute._id, case_name=case.display_name) }}" >
+            <i>
+            {% for hpo_gene in case.dynamic_gene_list %}
+              <input type="checkbox" name="dynamicGene" id="dynamicGene" value="{{hpo_gene.hgnc_id}}">
+              <input type="hidden" name="action" value="REMOVEGENES">
+              {{ hpo_gene.hgnc_symbol }}
+              &nbsp;&nbsp;
+            {% endfor %}
+            </i>
+          </form>
+        {% else %}
+          <span class="text-muted">No gene list generated</span>
+        {% endif %}
+      </div>
 
     {% if case.dynamic_gene_list %}
-      <div>
-        <div class="row d-flex justify-content-between mt-3 mb-3">
-          <div class="col align-content-center">
-            <div class="row mt-1">
-              {% if case.track == 'cancer' %}
-                <a class="btn btn-secondary btn-sm text-white"
-                  href="{{ url_for('variants.cancer_variants', institute_id=institute._id,
-                                   case_name=case.display_name, variant_type='clinical',
-                                   gene_panels=['hpo']) }}">
-                   Clinical HPO SNV variants
-                 </a>
-              {% else %}
-                <a class="btn btn-secondary btn-sm text-white"
-                  href="{{ url_for('variants.variants', institute_id=institute._id,
-                                   case_name=case.display_name, variant_type='clinical',
-                                   gene_panels=['hpo']) }}">
-                   Clinical HPO SNV variants
-                 </a>
-               {% endif %}
-            </div>
-            <div class="row mt-1">
-              <form action="{{ url_for('cases.update_clinical_filter_hpo', institute_id=institute._id, case_name=case.display_name)+'#hpo_clinical_filter' }}" method="POST">
-                <div class="form-check form-switch">
-                  <input type="checkbox" class="form-check-input" id="hpo_clinical_filter" onChange="this.form.submit()" name="hpo_clinical_filter"{% if case.hpo_clinical_filter %}checked{% endif %}>
-                  <label for="hpo_clinical_filter" class="form-check-label"><span data-bs-placement="top" data-bs-toggle='tooltip'
-                  title="Toggle on to use dynamic (HPO) panel instead of default gene panel for searching with clinical filter.">Use HPO list for clinical filter</span></label>
-                </div>
-              </form>
-            </div>
-
+      <div class="row d-flex justify-content-between">
+        <div class="col align-content-center"><!-- Edit synamic gene list-->
+          <div class="form-check form-switch">
```

Comment:
Weird, the switch doesn't stay selected on chrome and safari, which browser did you test with?

Reply:
https://user-images.githubusercontent.com/28093618/185359770-67353c00-79d0-4698-8596-711e902fdb11.mov

Reply:
Safari at least; might have been both - and the code looks that way too? 
Ah, actually it does stay selected in the clip as well! E.g. between 0:07 and 0:12 the state is weird to me: the user has ticked "Select all genes", which is now actively on, and then unticked a set. Its not the end of the world, but a simple button instead would be more logical to me. I'll drop by and point just since we are at the same place for once! 😆

Reply:
Fixed: 

https://user-images.githubusercontent.com/28093618/185393150-953f423a-f847-4c89-9ab5-b7df8d594235.mov